packages/post-rector/src/NodeAnalyzer/NetteInjectDetector.php

```diff
@@ -45,8 +45,9 @@ private function isInjectPropertyAlreadyInTheClass(Class_ $class): bool
             if ($phpDocInfo === null) {
                 continue;
             }
+            $phpDocInfoGetTagsByName = $phpDocInfo->getTagsByName('inject');
 
-            if ($phpDocInfo->getTagsByName('inject') === []) {
+            if ($phpDocInfoGetTagsByName === []) {
```

Comment:
It seems not handled by `$this->expectedNameResolver->resolveForCall()`, need a new tweak

Reply:
@TomasVotruba How can this be changing to : 

```php
$phpDocTagNode = $phpDocInfo->getTagsByName('inject');

# or better
$injectPhpDocTagNode = $phpDocInfo->getTagsByName('inject');
```

as you requested at https://github.com/rectorphp/rector/pull/4457#discussion_r508757259 ? The nearest probably change to:

```php
$injectphpDocInfoTagsName = $phpDocInfo->getTagsByName('inject');
```

By doing something like:

```php
            $get = str_ireplace('get', '', $arg0->value . $methodCallVarName . ucfirst($methodCallName));
            $by = str_ireplace('by', '', $get);

            return $by;
```

any suggestion? The `$this->expectedNameResolver->resolveForCall($methodCall)` returns null for that.

Reply:
@TomasVotruba I've updated to be renamed to: 

```diff
-            if ($phpDocInfo->getTagsByName('inject') === []) {
+            if ($injectphpDocInfoTagsName === []) {
```

As my commented above for nearest possible name.